### ISSUE TITLE
Enhancement of "Service we provide section"

### ISSUE THREAD

**Issue by kanak227** [PratikMane0112/TelMedSphere#24]
### What's the style idea?

We need to improve the navbar section and make it more user-friendly by enhancing its UI and adding more elements.
Kindly assign it to me for SWOC

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/PratikMane0112/TelMedSphere/blob/master/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a SWOC'25 contributor.
- [X] I am a KWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/PratikMane0112/TelMedSphere/blob/master.github/CODE_OF_CONDUCT.md)".

**Comment by PratikMane0112** [comment]
Hey @kanak227, WelCome to 💖TelMedSphere !!!!! Can you please tell more about enhancement you are thinking for navbar.
Please refer issue #19 #20 & #10.

**Comment by kanak227** [comment]
I guess the navbar enhancements I was thinking of are already assigned. 
In spite would like to work over the "Services we provide" section to make the cards responsive and working.
![Screenshot 2024-12-29 120441](https://github.com/user-attachments/assets/cf6f4801-c024-41b4-9ead-613d74a8cdc0)
Create different files for the 3 sections.

**Comment by PratikMane0112** [comment]
Hey @kanak227, please proceed with issue. Also give it a try, login with test accounts for better understanding of web app :
For patient -
username - patient@gmail.com
password - patient@gmail.com
For doctor -
username - doctor@gmail.com
password - doctor@gmail.com

Note : Do not forget to logout with accounts, when you are done ....

**Comment by PratikMane0112** [comment]
Also for any query or updates, join discord server (link in README file itself). Are you KWoC contributor too ? becoz you have also tick the KWoC'24 option while creating issue.



Note : You can start working on it now but as per SWoC guidelines I can only merge your PR after 1Jan

**Comment by kanak227** [comment]
Yes I am an KWOC contributor too . And as a query do I need to create a separate issue for working over this "services we provide" section issue

**Comment by PratikMane0112** [comment]
> Yes I am an KWOC contributor too . And as a query do I need to create a separate issue for working over this "services we provide" section issue

No need, you can edit the name of this issue

**Comment by kanak227** [comment]
Okay 👍

**Comment by PratikMane0112** [comment]
Hey @kanak227, please complete this issue in 2 days, else it will get closed

**Comment by kanak227** [comment]
Yeah sure
